src/libraries/System.Threading.RateLimiting/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Threading.RateLimiting.RateLimiterStatistics.set_CurrentAvailablePermits(System.Int64)</Target>
```

Comment:
The contract has a set but the implementation has an init property keyword. cc @BrennanConroy

Reply:
> The contract has a set

The ref assembly? I think that's because the msbuild task doesn't understand `init`?

Reply:
Yes the ref assembly. Either we would need to fix the existing CCI based GenAPI to respect init or wait until we have a roslyn based GenAPI which presumably will make this just work.

@BrennanConroy can you please submit a PR fixing that inconsistency? I believe that the set in the contract is incorrect as it's only init in the implementation.